src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildANetCoreApp.cs

```diff
@@ -40,6 +40,8 @@ public GivenThatWeWantToBuildANetCoreApp(ITestOutputHelper log) : base(log)
         [InlineData("netcoreapp1.1", null, "1.1.2", "1.1.2")]
         [InlineData("netcoreapp1.1", "1.1.0", "1.1.0", "1.1.0")]
         [InlineData("netcoreapp1.1.1", null, "1.1.1", "1.1.1")]
+        [InlineData("netcoreapp2.0", null, "2.0.0", "2.0.0")]
```

Comment:
@nguerrera @dsplaisted These 2 test should be good to have. But i will only break when stage0 has 2.1.1